chsdi/models/vector/uvek.py

```diff
@@ -2162,6 +2162,29 @@ class Kernkraftwerke(Base, Vector):
 register('ch.bfe.kernkraftwerke', Kernkraftwerke)
 
 
+class Kehrichtverbrennungsanlagen(Base, Vector):
+    __tablename__ = 'kehrichtverbrennungsanlagen'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/kehrichtverbrennungsanlagen.mako'
+    __bodId__ = 'ch.bfe.kehrichtverbrennungsanlagen'
+    __extended_info__ = True
+    id = Column('bgdi_id', Integer, primary_key=True)
+    wasteincinerationplantr = Column('wasteincinerationplantr', Unicode)
+    xtf_id = Column('xtf_id', Unicode)
+    number_id = Column('number_id', Integer)
+    name = Column('name', Unicode)
+    place = Column('place', Unicode)
+    beginningofoperation = Column('beginningofoperation', Integer)
+    web = Column('web', Unicode)
+    year = Column('year', Integer)
+    recycledwaste = Column('recycledwaste', Float)
```

Comment:
do we really need float? the data has only integer values for ``recycledwaste`` and ``electricity``?

Reply:
maybe the data gets rounded somewhere in the fme workbench?

Reply:
I think that in the future, some data would be in float. I propose to keep like this